h2o-genmodel/src/main/java/hex/genmodel/algos/tree/SharedTreeNode.java

```diff
@@ -443,6 +440,35 @@ void printDotEdges(PrintStream os, int maxLevelsToPrintPerEdge, float totalWeigh
     }
   }
 
+  public Map<String, Object> toJson() {
+    Map<String, Object> json = new HashMap<>();
+    json.put("nodeNumber", nodeNumber);
```

Comment:
is the bitset iself included? should the "json" representation be less raw and geared towards the visualization? eg. include what categorical values go which way?

Reply:
no, see domainValues handling below